### PR TITLE
upgrading sample-apiserver to 1.9

### LINKED ISSUE #63622: Flaky Test : [sig-api-machinery] Aggregator Should be able to support the 1.7 Sample API Server using the current Aggregator fails on "pull-kubernetes-e2e-gce" and "ci-kubernetes-e2e-gci-gce"
## Flaky Tests
* [pull-kubernetes-e2e-gce](https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/pr-logs/pull/59284/pull-kubernetes-e2e-gce/)
* [ci-kubernetes-e2e-gci-gce](https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/)

## Flakiest test 
[sig-api-machinery] Aggregator Should be able to support the 1.7 Sample API Server using the current Aggregator

## Failures in TestGrid
* [pull-kubernetes-e2e-gce](https://k8s-testgrid.appspot.com/presubmits-kubernetes-blocking#pull-kubernetes-e2e-gce&include-filter-by-regex=Aggregator%20Should%20be%20able%20to%20support%20th)
* [ci-kubernetes-e2e-gci-gce](https://k8s-testgrid.appspot.com/sig-api-machinery#gce)

## Triage link
More nuanced logs and details can be found at https://storage.googleapis.com/k8s-gubernator/triage/index.html?sig=api-machinery&job=ci-kubernetes-e2e-gci-gce&test=Aggregator%20Should%20be%20able%20to%20support%20the%201.7%20Sample%20AP

# Error
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/apimachinery/aggregator.go:76
gave up waiting for apiservice wardle to come up successfully
Expected error:
    <*errors.errorString | 0xc420131150>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
not to have occurred
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/apimachinery/aggregator.go:337

/priority failing-test
/priority important-soon
/kind flake
/sig-api-machinery
@kubernetes/sig-api-machinery-bugs 
/assign @liggitt 

@liggitt can you please help triage this issue?

### PR DESCRIPTION
**Pull request by fedebongio**
**What this PR does / why we need it**:
It upgrades the parameters on the sample-apiserver generation scripts to generate a 1.9 version of the sample api sever

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #63622

**Special notes for your reviewer**:
Let me know if I'm missing anything

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ixdy** [comment]
as noted in #68300, I'm a little worried that we're updating to 1.9 right as it's about to fall out of the support window, but I guess that's still better than 1.7 (which is over a year old now).

/approve

File: test/images/sample-apiserver/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.0
+1.1
```
Comment (ixdy) [inline test/images/sample-apiserver/VERSION]: it might be more intuitive to match the k8s api server version (i.e. use 1.9 here).

**Comment by ixdy** [comment]
also it looks like your git configuration might not be set up quite right - your author email is showing up using your laptop's hostname, rather than your real email address.

File: test/images/sample-apiserver/VERSION (hunk 1 above)
Comment (fedebongio) [reply test/images/sample-apiserver/VERSION]: I love the suggestion, will change it - I think I was stuck with the 1.7 being 1.0, and couldnt think about this, thank you!

**Comment by fedebongio** [comment]
> also it looks like your git configuration might not be set up quite right - your author email is showing up using your laptop's hostname, rather than your real email address.

**Comment by fedebongio** [comment]
@ixdy hopefully I fixed my user configuration, sorry about that

**Comment by fedebongio** [comment]
Closing in favor of a newer one to update sample api server to 1.10 (1.9 became too old once 1.12 was released)  #69233

**Pull request closed without merging**